client/src/components/generic/ConfirmationDialog/ConfirmationDialog.story.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import Dialog from "./ConfirmationDialog"
+const meta: Meta<typeof Dialog> = {
+  component: Dialog,
+  argTypes: {
+    variant: {
+      control: { type: "radio" },
+      options: ["default"]
+    },
+    text: {
+      name: "text for description"
+    }
+  }
+}
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const ConfirmationDialog: Story = {
```

Comment:
nit: we should have a story showing how to use on clicks

Reply:
done